### PR TITLE
[Doc] update docs for deploying starrocks on aws

### PR DESCRIPTION
**Pull request by hellolilyliuyi**
## What type of PR is this:
- [ ] BugFix
- [ ] Feature
- [ ] Enhancement
- [ ] Refactor
- [ ] UT
- [x] Doc
- [ ] Tool

## Which issues of this PR fixes:

Fixes #

## Checklist:

- [ ] I have added test cases for my bug fix or my new feature
- [ ] This pr will affect users' behaviors
- [ ] This pr needs user documentation (for new or modified features or behaviors)
  - [ ] I have added documentation for my new feature or new function

## Bugfix cherry-pick branch check:
- [x] I have checked the version labels which the pr will be auto backported to target branch
  - [x] 3.0
  - [x] 2.5
  - [ ] 2.4
  - [ ] 2.3

### CODE REVIEW

File: docs/administration/starrocks_on_aws.md [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+   1. Install MySQL on the EC2 Bastion Host.
+
+   2. Use the following command to connect the StarRocks cluster:
+
+      ```Bash
+      mysql -u root -h 10.0.xx.xx -P 9030 -p
+      ```
+
+      - host:
+        You can find FE's private IP address according to the following steps:
```
Comment (amber-create) [inline docs/administration/starrocks_on_aws.md:110]: image
-> the FE's private IP address 或者 the private IP address of the FE

File: docs/faq/loading/Routine_load_faq.md [hunk 2]
```diff
@@ -1,27 +1,81 @@
 # Routine Load
 
-## Does mysql binlog data produced by kafka count as text format data?
+## How can I improve loading performance?
 
-When imported into kafka via canal, it is in json format.
+**Method 1: Increase the actual load task** **parallelism** by splitting a load job into as many parallel load tasks as possible.
```
Comment (amber-create) [inline docs/faq/loading/Routine_load_faq.md:5]: image
加粗格式 ** 有点问题

File: docs/faq/loading/Routine_load_faq.md [hunk 3]
```diff
@@ -1,27 +1,81 @@
...
+- `max_routine_load_task_concurrent_num`: the default maximum task parallelism for a Routine Load job. The default value is `5`. This parameter is a an FE dynamic parameter. For more information and the configuration method, see [Parameter configuration](../../administration/Configuration.md#loading-and-unloading).
+
+Therefore, when the number of partitions to be consumed and the number of BE nodes alive are greater than the other two parameters, you can increase the values of `desired_concurrent_number` and `max_routine_load_task_concurrent_num` parameters to increase the actual load task parallelism.
+
+For example, the number of partitions to be consumed is `7`, the number of live BE nodes is `5`, and `max_routine_load_task_concurrent_num` is the default value `5`. At this time, if you need to increase the load task parallelism to the upper limit, you need to set `desired_concurrent_number` to `5` (the default value is `3`). Then, the actual task parallelism `min(5,7,5,5)`is computed to be `5`.
```
Comment (amber-create) [inline docs/faq/loading/Routine_load_faq.md:28]: image
中间貌似少空格

File: docs/faq/loading/Routine_load_faq.md [hunk 4]
```diff
@@ -1,27 +1,81 @@
...
+
+For more parameter descriptions, see [CREATE ROUTINE LOAD](../../sql-reference/sql-statements/data-manipulation/ROUTINE%20LOAD.md#example).
 
-**Issue description:**
+**Method 2:** **Increase the amount of data consumed by a Routine Load task from one or more partitions.**
```
Comment (amber-create) [inline docs/faq/loading/Routine_load_faq.md:32]: image
这块加粗格式 ** 看起来也有点问题

File: docs/faq/loading/Routine_load_faq.md [hunk 5]
```diff
@@ -1,27 +1,81 @@
...
 ```
 
-**Cause:**
+Normally, the field `left_bytes` in the log is greater than or equal to `0`, indicating that the amount of data consumed by a load task has not exceeded `max_routine_load_batch_size` within `routine_load_task_consume_second`. This means that a batch of scheduled load tasks can consume all data from Kafka without delay in consumption. In this scenario, you can set a larger value for `routine_load_task_consume_second` to increase the amount of data consumed by a load task from one or more partitions.
+
+If the field `left_bytes` is less than `0`, it means that the amount of data consumed by a load task has reached `max_routine_load_batch_size` within `routine_load_task_consume_second`. Every time data from Kafka fills the batch of scheduled load tasks. Therefore, it is highly likely that there is remaining data that has not been consumed in Kafka, causing delay in consumption. In this case, you can set a larger value for `max_routine_load_batch_size`.
+
+## What do I do if the result of SHOW ROUTINE LOAD shows that the load job is in the `PAUSED` state?
+
+- Check the field `ReasonOfStateChanged` and it reports the error message `Broker: Offset out of range`.
```
Comment (amber-create) [inline docs/faq/loading/Routine_load_faq.md:52]: image
-> and see if it reports?

File: docs/faq/loading/Routine_load_faq.md [hunk 6]
```diff
@@ -1,27 +1,81 @@
...
+
+  - If it is caused by incorrect data format in the data source, you need to check the data format and fix the issue. After successfully fixing the issue, you can use [RESUME ROUTINE LOAD](../../sql-reference/sql-statements/data-manipulation/RESUME%20ROUTINE%20LOAD.md) to resume the paused load job.
+
+  - If it is because that StarRocks cannot parse the data format in the data source, you need to adjust the threshold `max_error_number`. You can first execute [SHOW ROUTINE LOAD](../../sql-reference/sql-statements/data-manipulation/SHOW%20ROUTINE%20LOAD.md) to view the value of `max_error_number`, and then use [ALTER ROUTINE LOAD](../../sql-reference/sql-statements/data-manipulation/alter-routine-load.md) to increase the threshold. After modifying the threshold, you can use [RESUME ROUTINE LOAD](../../sql-reference/sql-statements/data-manipulation/RESUME%20ROUTINE%20LOAD.md) to resume the paused load job.
+
+## What do I do if the result of SHOW ROUTINE LOAD shows that the load job is in the `CANCELLED` state?
+
+  **Cause analysis:** The load task encountered an exception during loading, such as the table deleted.
```
Comment (amber-create) [inline docs/faq/loading/Routine_load_faq.md:73]: image
load task 应该是 load job？

File: docs/faq/loading/Routine_load_faq.md [hunk 7]
```diff
@@ -1,27 +1,81 @@
...
+  **Solution:** When troubleshooting and fixing the issue, you can refer to the error messages in the fields `ReasonOfStateChanged` and `ErrorLogUrls`. However, after fixing the issue, you cannot resume the cancelled load job.
 
-The current librdkafka does not support sasl authentication.
+## Can Routine Load guarantee consistency semantics when consuming from Kafka and writing to StarRocks?
 
-**Solution:**
+   Routine Load guarantees exactly-once semantics.
 
-Compile librdkafka again. Please refer to 'compile librdkafka'.
+   Each load task is a individual transaction. If an error occurs during the execution of the transaction, the transaction is aborted, and the FE does not update the consumption progress of the relevant partitions of the load tasks. When FE schedules the load tasks from the task queue next time, the load tasks send the consumption request from the last saved consumption position of the partitions, thus ensuring exactly-once semantics.
```
Comment (amber-create) [inline docs/faq/loading/Routine_load_faq.md:81]: image
前面漏了冠词 the

**Review by amber-create — APPROVED** [review]

**Review by EsoragotoSpirit — APPROVED** [review]

**Comment by hellolilyliuyi** [comment]
@mergify backport branch-3.0 branch-2.5 branch-2.4 branch-2.3

**Pull request merged**